src/Symfony/Component/Validator/Validator/RecursiveContextualValidator.php

```diff
@@ -723,7 +724,12 @@ private function validateInGroup(mixed $value, ?string $cacheKey, MetadataInterf
     {
         $context->setGroup($group);
 
-        foreach ($metadata->findConstraints($group) as $constraint) {
+        $constraints = $metadata->findConstraints($group);
+        if ($this->autoSequenceConstraints && 1 < \count($constraints)) {
+            $constraints = [new Sequentially($constraints)];
```

Comment:
wouldn't this invalidate the cache we have below due to having a new constraint each time ?